src/style.css

```diff
@@ -1 +1,30 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You are using the `body` tag for styling, which is not recommended according to the checklist. Instead, consider creating a class for the body and apply the styles to that class.